<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serial</class>
 <widget class="QWidget" name="serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>游戏进行中...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/new/picture/1.ico</normaloff>:/new/picture/1.ico</iconset>
  </property>
  <widget class="QPushButton" name="send1Button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>获取身份</string>
   </property>
  </widget>
  <widget class="QLabel" name="gamer">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="plangamer">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="picture_right">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/picture/1.JPG);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startgame">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send3Button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>是否卧底</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>再来一次</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="picture_left">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/picture/1.JPG);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>18</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>18</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>20</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>com1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>com2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>com3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>com4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>com5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>com6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>com7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>com8</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>游戏总人数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>参与人数：</string>
   </property>
  </widget>
  <zorder>picture_left</zorder>
  <zorder>send1Button</zorder>
  <zorder>gamer</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>plangamer</zorder>
  <zorder>picture_right</zorder>
  <zorder>returnButton</zorder>
  <zorder>startgame</zorder>
  <zorder>send3Button</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
